examples/sockets/TCP.cpp

```diff
@@ -34,17 +35,17 @@ void runTcpServer(unsigned short port)
     std::cout << "Client connected: " << socket.getRemoteAddress().value() << std::endl;
 
     // Send a message to the connected client
-    const char out[] = "Hi, I'm the server";
-    if (socket.send(out, sizeof(out)) != sf::Socket::Status::Done)
+    static constexpr std::string_view out = "Hi, I'm the server";
```

Comment:
Is `static` really necessary?

Reply:
Does the compiler not already optimize this?

Reply:
I'm not sure. In some cases it may be smart enough to do it. I'm really not sure how to check that without looking at the generated code.